app/services/publisher_statement_getter.rb

```diff
@@ -40,6 +40,8 @@ def replace_account_identifiers_with_titles(transactions)
       account_identifier = transaction["channel"]
       if account_identifier.starts_with?(Publisher::OWNER_PREFIX)
         transaction["channel"] = "All"
+      elsif account_identifier.blank?
+        transaction["channel"] = "Manual"
```

Comment:
Consider 

```ruby
  transaction["channel"] = PotentialPayment::MANUAL
```

Reply:
"Manual" is a label from the transactions table.